packages/shared/src/dto/workflows/step.dto.ts

```diff
@@ -2,7 +2,7 @@ import type { JSONSchemaDto } from './json-schema-dto';
 import { Slug, StepTypeEnum, WorkflowOriginEnum } from '../../types';
 import { StepContentIssueEnum, StepIssueEnum } from './step-content-issue.enum';
 
-export type StepDataDto = {
+export type StepResponseDto = {
```

Comment:
if we used this param as request DTO, my guess is that it is not self-explanatory enough